tests/foreman/api/test_user.py

```diff
@@ -372,3 +372,167 @@ def test_positive_update(self):
                 # pylint:disable=no-member
                 set([role.id for role in chosen_roles] + [self.anon_role.id]),
             )
+
+
+class SshKeyInUserTestCase(APITestCase):
+    """Implements the SSH Key in User Tests"""
+
+    @stubbed
+    @tier1
+    def test_positive_create_ssh_key(self):
+        """SSH Key can be added to User
+
+        :id: d00905f6-3a70-4e2f-a5ae-fcac18274bb7
+
+        :steps:
+
+            1. Create new user with all the details
+            2. Add SSH key to the above user
+
+        :expectedresults: SSH key should be added to user
+
+        :CaseImportance: Critical
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_create_ssh_key_super_admin(self):
+        """SSH Key can be added to Super Admin user
+
+        :id: 397eea22-759c-4cd4-bda1-0e7835566c72
+
+        :expectedresults: SSH Key should be added to Super Admin user
+
+        :CaseImportance: Critical
+        """
+
+    @stubbed
+    @tier1
+    def test_negative_create_ssh_key(self):
+        """Invalid ssh key can not be added in User Template
+
+        :id: e924ff03-8b2c-4ab9-a054-ea491413e143
+
+        :steps:
+
+            1. Create new user with all the details
+            2. Attempt to add invalid string as SSH Key to above user
+                e.g blabla
+
+        :expectedresults:
+
+            1. Invalid SSH key should not be added in user
+            2. Satellite returns 'Fingerprint could not be generated' error
+
+        :CaseImportance: Critical
+        """
+
+    @stubbed
+    @tier1
+    def test_negative_create_invalid_length_ssh_key(self):
+        """Attempt to add SSH key that has invalid length
+
+        :id: 899f0c46-c7fe-4610-80f1-1add4a9cbc26
+
+        :steps:
+
+            1. Create new user with all the details
+            2. Attempt to add invalid length of SSH Key to above user
+
+        :expectedresults: Satellite should raise 'Length could not be
+            calculated' error
+
+        :CaseImportance: Critical
+        """
+
+    @stubbed
+    @tier3
```

Comment:
doesn't sound as tier 3 test case. What system components are used here? Only user and ssh keys as in previous tier1 test cases